src/integrationTest/resources/application.yml

```diff
@@ -1,10 +1,13 @@
 spring:
-  flyway.locations: classpath:db/migration
+  flyway:
+    schemas: DBREFDATA
```

Comment:
Just to double check, this is the only instance in caps, is that correct?

Reply:
flyway users schema in caps